ui/components/src/ThresholdsEditor/ThresholdsEditor.tsx

```diff
@@ -135,12 +137,15 @@ export function ThresholdsEditor({ thresholds, onChange }: ThresholdsEditorProps
   };
 
   const handleModeChange = (event: React.MouseEvent, value: string): void => {
+    const mode = value === 'Percent' ? 'Percent' : undefined;
```

Comment:
Would we want the mode to be `Absolute` if it's not `Percent`?

Reply:
Since the default is `Absolute`, setting it `undefined` here keeps the json schema short and simple